perf/perf_test.py

```diff
@@ -71,6 +71,15 @@ def setUp(self):
             else:
                 self.cancel("Install the package for perf supported\
                           by %s" % detected_distro.name)
+        if run_type == 'upstream':
+            if 'rhel' in detected_distro.name:
+                deps.extend(['systemtap-sdt-devel', 'slang-devel',
+                             'perl-ExtUtils-Embed', 'libcap-devel',
+                             'numactl-devel', 'libbabeltrace-devel',
+                             'libpfm-devel', 'java-1.8.0-openjdk-devel',
+                             'libtraceevent-devel'])
```

Comment:
can we add package dependency for suse as well

Reply:
So it turns out that some of the dependent packages are not available with SUSE, at least I could not locate them. For now I have added a cancel() call if upstream is the target with SUSE as a base

```
 TODO I could only locate systemtap-sdt-devel, libunwind-devel,
 slang-devel, and libcap-devel packages. Missing list includes
 elfutils-devel, libbabeltrace-devel, java-1.8.0-openjdk-devel
 and libpfm4-devel. For now cancel the test due to missing
 dependent packages
```